CloverSDK.podspec

```diff
@@ -0,0 +1,55 @@
+Pod::Spec.new do |s|
+  s.name         = "CloverSDK"
+  s.version      = "0.3.0"
+  s.summary      = "Clover SDK"
+  s.homepage     = "https://www.clover.com/features/integrations"
+  s.license      = { :type => "MIT", :file => "LICENSE" }
```

Comment:
@mmaietta, I changed it to Apache, and I found our license file in the clover-android-sdk repo, but the first line of the license file contains some string interpolation:
`Copyright (C) ${year} ${name}`
Is it supposed to be that way? If so, do you know where are these values being filled?